app/test_main.py

```diff
@@ -15,6 +15,17 @@
         (27, 27, [2, 2]),
         (28, 28, [3, 2]),
         (100, 100, [21, 17])
+    ],
+    ids=[
+        "should return correct result",
+        "should return correct result",
+        "should return correct result",
+        "should return correct result",
+        "should return correct result",
+        "should return correct result",
+        "should return correct result",
+        "should return correct result",
+        "should return correct result"
```

Comment:
Mb a little bit more descriptive ids names?)
Can't get fast which test testes which case, just try to make it according to best cases)